pycharm-community.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015 Petr Hracek <phracek@redhat.com> -->
+<application>
+ <id type="desktop">pycharm-community.desktop</id>
+ <licence>CC-BY-SA</licence>
+ <name>PyCharm</name>
+ <summary>Pycharm - The most interlligent Python IDE</summary>
+ <description>
+  <p>
+      PyCharm’s smart code editor provides first-class support for Python, JavaScript, CoffeeScript, TypeScript, CSS, popular template languages and more. Take advantage of language-aware code completion, error detection, and on-the-fly code fixes!
+  </p>
+ </description>
+ <screenshots>
+   <screenshot type="default" width="1968" height="814">
+       http://www.jetbrains.com/pycharm/img/40/autocomplete@2x.png
+   </screenshot>
+   <screenshot width="642" height="690">
+       http://www.jetbrains.com/pycharm/img/40/scr_why_smartassist.png
+   </screenshot>
+ </screenshots>
+ <url type="homepage">http://www.jetbrains.com/pycharm/</url>
+ <updatecontact>phracek@redhat.com</updatecontact>
```

Comment:
If possible we'd love to see releases tags and [OARS](https://odrs.gnome.org/oars) metadata (although that will basically all say 'it's fine'!) but it's not a hard requirement.

You may also want to link some external icons in here - similar to the Spotify app, since otherwise it's unlikely to show up in GNOME Software: https://github.com/flathub/com.spotify.Client/blob/master/com.spotify.Client.appdata.xml#L11

Reply:
It repackages upstream tarball (and doesn't use extra data) so icon will show up just fine.

Reply:
Of course, being very slow this afternoon - apologies.